crates/experimental-api/src/exe/err.rs

```diff
@@ -24,6 +25,8 @@ pub enum Error {
     ),
     #[error("Program error: {0:?}")]
     Program(#[from] cairo_vm::types::errors::program_errors::ProgramError),
+    #[error("Sierra compilation error: {0:?}")]
```

Comment:
This comment is unrelated to this PR, but what do you think about changing the error messages and making them start with lowercase instead of uppercase? In other parts of the code base, we also use error messages starting with lowercase, so that would make it consistent.

Reply:
https://github.com/eigerco/beerus/issues/636

Reply:
https://github.com/eigerco/beerus/pull/637